src/manage_sql.c

```diff
@@ -57061,24 +57102,29 @@ tag_add_resources_list (tag_t tag, const char *type, array_t *uuids,
   gchar *resource_permission, *current_uuid;
   int index;
 
+  gchar *resource_type = g_strdup (type);
+
   if (type_is_info_subtype (type))
     resource_permission = g_strdup ("get_info");
   else if (type_is_asset_subtype (type))
     resource_permission = g_strdup ("get_assets");
   else if (type_is_report_subtype (type))
     {
       resource_permission = g_strdup ("get_reports");
-      type = g_strdup("report");
+      g_free (resource_type);
+      resource_type = g_strdup("report");
```

Comment:
```suggestion
      resource_type = g_strdup ("report");
```
Would be great if the CI would catch these.